### DISCUSSION TITLE
Anyone made it work on windows?

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [jax-ml/jax#15712] [Q&A]
I want to test jax whisper but it requires latest jax version which I couldn't find a way to install

The only one I found for windows is 0.3.7 and it is not working

**Comment by adam-hartshorne** [comment]
This has a lot of wheels of JAX (up to when there were a number of issues with other dependencies)

https://github.com/cloudhan/jax-windows-builder

I personally got v0.4.4 to build using the advice here, https://github.com/google/jax/issues/14369 , but its quite painful and a time consuming exercise. As far as I know, since then it is broken in a way that isn't fixable without 3rd party libraries which JAX relies on changing their code to be windows compatible.

**Comment by FabiJa** [comment]
I have also problems building Jax on Windows, probably because being a noob.

After issuing the command `python .\build\build.py --enable_cuda --cuda_path="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8" --cudnn_path="C:\Program Files\NVIDIA\CUDNN\v8.7" --cuda_version="11.8" --cudnn_version="8.7"`

I get 
```
Bazel binary path: .\bazel-6.1.2-windows-x86_64.exe
Bazel version: 6.1.2
Python binary path: ./.conda/envs/pyfdi/python.exe
Python version: 3.9
NumPy version: 1.24.4
MKL-DNN enabled: yes
Target CPU: AMD64
Target CPU features: release
CUDA enabled: yes
CUDA toolkit path: C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8
CUDNN library path: C:\Program Files\NVIDIA\CUDNN\v8.7
CUDA version: 11.8
CUDNN version: 8.7
NCCL enabled: yes
TPU enabled: no
ROCm enabled: no
```

I finally get an error during the process with the following error message (filtered for the most meaningful...):
```
ERROR: ./jax/WORKSPACE:15:15: fetching cuda_configure rule //external:local_config_cuda
ERROR: Skipping '@xla//xla/python:enable_gpu': no such package '@local_config_cuda//cuda': Repository command failed
ERROR: @xla//xla/python:enable_gpu :: Error loading option @xla//xla/python:enable_gpu: no such package '@local_config_cuda//cuda': Repository command failed
```
Is this due to an connection issue ( I am behind a proxy) or what is the problem with this `xla` package?
Any advise?
